build.yaml

```diff
@@ -1,14 +1,14 @@
 ---
 name: "NextPVR"
-guid: "9574ac10-bf23-49bc-949f-924f23cfa48f"
-version: "4.0.0.0"
+guid: "62c5a9c3-a283-4379-8756-7ea6ed3e7085"
```

Comment:
Any particular reason for why the GUID is getting changed?

Reply:
The old plug-in can be installed side by side with this one.  Several v4 users haven't upgraded and some use both on different ports.

Reply:
OK that has me confused - I thought this was an API-compatible rewrite, in which case it would entirely replace the existing plugin, drop-in, with no usability changes and no reason to keep using v4. If they're expected to run side-by-side then that complicates things, though people can also always just install V4 from the repo too.

I guess we can continue with this PR and just leave V4 in the plugin repo but while also encouraging a move to this one. If anything changes that needs user input this will have to be documented, preferably in at least two of: README, A Blog Post, a Reddit Thread, Documentation.

Reply:
Very few new users will use v4 and I am not sure if there are any v4 users we dont see any support posts for v4 jellyfin users on the nextpvr forum.  I don't know how your CI works but I know I can contribute to one repo with two active branches for Kodi. I guess the owner of the v4 branch could answer what they want.

Reply:
@joshuaboniface I could compile a v4 branch build for existing users "on request" at the NextPVR forums if the old version needs a rebuild.  That way we can find out if there are actual users.  I don't think it should be available for download for new users.

Reply:
@joshuaboniface @crobibero @dkanada @oddstr13  Can I get an update on the review please.  This rewritten PR was supposed to make the review process from the original PR I submitted two months ago a little easier but It didn't seem to speed the process up.